src/applications/facility-locator/sass/facility-locator.scss

```diff
@@ -620,10 +620,16 @@
   display: inline-block;
   font-size: 1.2rem;
   background-image: none;
+  background-color: rgba(0,0,0,0);
```

Comment:
hmm... do we already have a var for this value? or a standard way to make a transparent background?

Reply:
Based on existing code we normally just to `transparent`. But maybe there's a reason to do it as an `rgba` value that I'm not aware of?

Reply:
No, just my lack of knowledge with CSS shorthands 😄 
(was the only thing that came to mind to set alpha levels)